docs/sphinx/_toc.yml.in

```diff
@@ -2,13 +2,18 @@ defaults:
   numbered: False
 root: index
 subtrees:
-- entries: 
+- caption: Installation
```

Comment:
Why did we change this to caption, Randy?

Reply:
Just following the rocDecode structure as recommended.